doc/Command-Reference.md

```diff
@@ -2348,6 +2348,74 @@ This command is used to delete a configured DHCP Relay Destination IP address or
   Restarting DHCP relay service...
   ```
 
+**config dhcp_relay ipv4 helper add/del**
+
+This command is used to add or delete an IPv4 DHCP Relay helper IP address or multiple IP addresses to a VLAN. Note that more than one DHCP Relay helper IP address can be operated on a VLAN interface.
```

Comment:
Suggest: This command is used to add or delete IPv4 DHCP Relay helper addresses to a VLAN. ...

Reply:
Thanks for suggesting, fixed